modules/Bio/EnsEMBL/Compara/PipeConfig/Metazoa/Synteny_conf.pm

```diff
@@ -0,0 +1,63 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::Metazoa::Synteny_conf
+
+=head1 DESCRIPTION
+
+This is the EG Metazoa specific version of the general Bio::EnsEMBL::Compara::PipeConfig::Synteny_conf
+
+Example: init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::Metazoa::Synteny_conf  -pipeline_name <> -ptree_db/alignment_db <>
+
+#init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::EBI::Metazoa::Synteny_conf -pipeline_name synteny_metazoa_42_95 -host mysql-ens-compara-prod-5 -port 4615
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+package Bio::EnsEMBL::Compara::PipeConfig::Metazoa::Synteny_conf;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Hive::Version 2.4;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+
+use base ('Bio::EnsEMBL::Compara::PipeConfig::Synteny_conf');  # All Hive databases configuration files should inherit from HiveGeneric, directly or indirectly
+
+sub default_options {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::default_options},   # inherit the generic ones
+        'division'     => 'metazoa',
+        
+        'alignment_db'    => 'compara_curr',
```

Comment:
These db parameters are present as default in the `Bio::EnsEMBL::Compara::PipeConfig::Synteny_conf` for `compara_curr`. The only non-default value provided here is `'division'     => 'metazoa'` which could be provided on the command line:
`init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Synteny_conf -host mysql-ens-compara-prod-X -port XXXX         -division metazoa`
I am not sure that this PR is a very necessary addition :slightly_smiling_face:

Reply:
I assume, we can drop  PipeConfig/Metazoa/Lastz_conf.pm,  as well, as most of the PipeConfig/Plants, for homogenity.

Reply:
As far as I can tell `Bio::EnsEMBL::Compara::PipeConfig::Metazoa::Lastz_conf` and many others of the division specific configs contain more than the default parameters in `Bio::EnsEMBL::Compara::PipeConfig::*_conf`, and analysis tweaks that are very much needed. We use environmental variables with the pipelines for division `$COMPARA_DIV` in the `init_pipeline` command line, so for our side, we wouldn't need a conf just for the division parameter. I wouldn't say we have to drop this division specific config, but at the moment, the two db parameters at least are not needed in it. You could keep this config, with the future implications that other parameters may change or analysis tweaks are likely to be added?

Reply:
I see. Thanks a lot, Cristi. Actually, I prefer specifying init options. And, yes, config is not that deverged from the base one. So, if we don't need this config as a marker that Synteny should be run for metazoan compara, I'll close this PR.

Reply:
discussed with @CristiGuijarro . devision specific PipeConfig dirs are not used as any kind of a marker. thus closing this PR. thanks.